ptu/global.ini

```diff
@@ -44492,14 +44504,14 @@ ea_ui_frontend_specialevent_PlayerBirthday_Description,P=Player Birthday, text ,
 ea_ui_frontend_specialevent_PlayerBirthday_Title,P=Player Birthday
 ea_ui_frontend_specialevent_StPatricks_Description,P=Finde die versteckten Töpfe überall auf der Karte.
 ea_ui_frontend_specialevent_StPatricks_Title,P=St. Patricks
-ea_ui_frontend_specialevent_ValentinesDay_Description,P=Bilde ein Duo mit deinem Partner, um gegen andere anzutreten.
+ea_ui_frontend_specialevent_ValentinesDay_Description=Verbreite die Liebe dieses Lunar New Year mit besonderen Coramor- und Red Festival-Belohnungen! Setze dich in einer Runde Kill Collector (FPS oder Fahrzeug) an die Spitze, um eine glücksverheißende Year of the Dog Coin zu verdienen, oder gewinne den Duo Showdown, um eine Coramor Coin zu erhalten, die sicherlich die Herzen von dir und deinem Partner ein wenig schneller schlagen lässt.
```

Comment:
Hier ist die falsche Übersertzung von ea_ui_frontend_specialevent_LunarNewYear_Description reingerutscht

Reply:
Hey. Der Text der beiden Variablen ist identisch in der englischen Datei ^^